.github/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+### Description:
+
+<!--
+This should be a brief description of the PR. Details should be contained in commit messages
+
+The PR should be restricted to only two or three changes. For adding new dashboards this should be done over more than one PR due to the size of the JSON files for dashboards.
+
+-->
+
+### Additional Notes
+
+<!-- This section can be removed with not required 
+
+Examples of summary of changes:
+- Which dashboards were updated
+- If new dashboards were created, a summary of what the dashboards will show
+-->
```

Comment:
This should go into the section above, and should be required as you can have it where someone gets two branches confused :thinking: